identity-and-access-management/audit-trail/reference-content/product-integration-with-adt.mdx

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  title: Audit Trail resource integration
+  description: Discover current and planned Scaleway resource integrations with Audit Trail for event tracking.
+content:
+  h1: Audit Trail resource integration
+  paragraph: Discover current and planned Scaleway resource integrations with Audit Trail for event tracking.
+tags: audit-trail product-integration scaleway-resources
+dates:
+  validation: 2024-11-26
+  posted: 2024-11-26
+categories:
+  - identity-and-access-management
+---
+
+The following table provides details about the Scaleway resources that are integrated with Audit Trail.
+
+## Resource integration available
+
+| **Product Name**           | **Audit Trail Integration**     |
+|----------------------------|---------------------------------|
+| Secret Manager             | **Integrated**                  |
+| Kubernetes                 | **Integrated**                  |
+
+## Resource integration planned
+
+The following table provides details about the Scaleway resources that will be integrated with Audit Trail.
+
+
+| **Product Name**           | **Audit Trail Integration**     |
+|----------------------------|---------------------------------|
+| IAM                        | **Planned**                     |
+| Object Storage             | **Planned**                     |
+| Billing                    | **Planned**                 |
```

Comment:
Can you remove the "Planned"? it implies that others are not planned and I do not want our users to think that